Learning Paths/dot_net_backend_developer.md

```diff
@@ -42,7 +42,7 @@ In progress...
 
 🎓 .NET Backend Developer - Courses / Certification
 -----
-In progress...
+[Develop Web Apps with ASP.NET Core](https://learn.microsoft.com/en-us/collections/8mq4im72x6xprg)
```

Comment:
Please change it from [Develop Web Apps with ASP.NET Core](https://learn.microsoft.com/en-us/collections/8mq4im72x6xprg)
to
Develop Web Apps with ASP.NET Core - [link](https://learn.microsoft.com/en-us/collections/8mq4im72x6xprg)